egasub/submission/submittable/unaligned.py

```diff
@@ -18,6 +20,14 @@ def __init__(self, path):
         self._run.files = map(lambda file_: EFile.from_dict(file_), self.metadata.get('files'))
 
     @property
+    def files(self):
+        return self._run.files
+
+    @property
     def type(self):
         return self.__class__.__bases__[0].__name__.lower()
 
+    def ftp_files_remote_validate(self,host,username, password):
+        for _file in self._run.files:
+            if not file_exists(host,username,password,_file.file_name):
+                self._add_ftp_file_validation_error("run",self.run.alias,"fileName","File missing on FTP ega server: %s" % _file.file_name)
```

Comment:
only need two parameters here: `file_name`, and `error` (the text of error message)